### DISCUSSION TITLE
Questions about software that can be downloaded from this github

### DISCUSSION THREAD

**Discussion by maho-sk** [Ignitarium-Renesas/RZV2L_AiLibrary#3] [General]
I have several questions regarding the software available for download from GitHub.

・Is it possible to check the operation of the exe version immediately on the RZ/V2L smart board?

・How far does the source file other than exe include the driver part?
Is it all the features supported by the smart board or just the features used in the library?

・If I develop a new board and need a driver that is not included in the library, do I need to create a driver myself?

**Comment by mohitb-ign** [comment]
Q3 : Is it possible to check the operation of the exe version immediately on the RZ/V2L smart board?
Answer:  Yes. Operation of exe can be verified on RZV2L smarc board immediately. User needs to copy the application folder as it is on the board and run the executable.
Executable and weight files inside ‘exe’ folder is required to run on the board without any extra steps.
 
Q4:  How far does the source file other than exe include the driver part? Is it all the features supported by the smart board or just the features used in the library?
Answer:  The library is usable on RZV2L smarc boards. Executable contains only the application part and not the driver part.
 
Q5:  If I develop a new board and need a driver that is not included in the library, do I need to create a driver myself?
Answer: Driver is not scope of this application library. Please connect with us for additional discussion.
